Yoast/wordpress-seo
# Deprecate class opengraph

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* This PR deprecates the last functions in `class-opengraph`, and moves this class, as well as `class-twitter` and `class-opengraph-image` to the `deprecated` directory.
* We restored a filter in `class-twitter` that had been deleted without being deprecated. 
* The `get_instance()` function from `class-twitter` returns null, instead of `self::$instance`, since it otherwise would have given two deprecation notices.
* There is no replacement filter/action for the deprecated `do_action` and `apply_filters`, because we didn't see a one-to-one correspondence.

## Test instructions

This PR can be tested by following these steps:

* In the backend, create a post with a category and a tag 
* In the backend, set a `Facebook App ID` under `SEO` --> `Social` (for example `246554168145`).
* In the frontend, see meta tags `article:section`, `article:tag` and `fb:app_id`
* Check out this branch, and notice that the meta tags have disappeared.

## UI changes

## Documentation

## Quality assurance


Fixes #
